documentation/release_notes.rst

```diff
@@ -8,6 +8,21 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.1.1
+===============
+New Features
+------------
+- Improved ``sort`` algorithm performance for the arithmetic data types with ``std::less`` or ``std::greater`` comparison operator and DPC++ policy.
+- AOT compilation for CPU devices is now possible with the use of ``ONEDPL_USE_AOT_COMPILATION`` cmake option.
+
+Fixes Issues
+------------
+- Fixed ``Device name missing`` error occurring with the following cmake options: ``ONEDPL_USE_AOT_COMPILATION=ON``, ``ONEDPL_DEVICE_BACKEND=GPU`` or not specified and ``CMAKE_BUILD_TYPE`` is not specified.
```

Comment:
Because of the options, that also only affects the tests, right? If so, is it important enough to mention? If the impact is more than just our tests, we need to explain it more generally, not referring to Cmake macros.

Reply:
Omitted the notes about ONEDPL_USE_AOT_COMPILATION cmake option as was suggested.